target/linux/ramips/mt7621/config-5.4

```diff
@@ -281,6 +282,7 @@ CONFIG_SERIAL_8250_NR_UARTS=3
 CONFIG_SERIAL_8250_RUNTIME_UARTS=3
 CONFIG_SERIAL_MCTRL_GPIO=y
 CONFIG_SERIAL_OF_PLATFORM=y
+CONFIG_SFP=y
```

Comment:
`SFP` and `MDIO_I2C` can be built as modules

Reply:
> 
> 
> Hi, your first two commits lack commit description and Signed-off.
> 

I shall fix that.

> Despite, consider using "ramips:" prefix for your first patch, as this one will be in target directory.

OK.

Reply:
How do you add a module to the image?
Only mark it as a module seems not enough to get is in the image.
I see that the module is build but not added to the image.

Reply:
at803x needs to be build-in the kernel other wise it is loaded to late.
mt7530 is already loaded and initialized before at803x is loaded.

Maybe some other tricks to fix that?

Reply:
at803x needs to be built-in while others are built-in, yes.

But SFP and MDIO_I2C (which SFP depends on) can be built as modules.

Further more, we can build every network related part as a module, this can decrease kernel size even more, and it may fit in HC5962's 2M kernel partition.